### DISCUSSION TITLE
扩展复制粘贴，按文档来找不到对应的依赖。

### DISCUSSION THREAD

**Discussion by gangdiedao** [dream-num/univer#2849] [Q&A | 常见问题] [answered]
import { ISheetClipboardService, ISheetClipboardHook } from '@univerjs/sheet'
提示找不到 @univerjs/sheet

**Comment by hexf00** [comment, marked as answer]
Hi! It seems you might need to import from `@univerjs/sheets-ui` instead of `@univerjs/sheet`. Please try updating your import statement and see if that resolves the issue. https://univer.ai/typedoc/@univerjs/sheets-ui/interfaces/ISheetClipboardService

**Reply by gangdiedao** [reply]
可以了，谢谢
